<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-H8GD1N65" failures="1" tests="4" name="Testng.firstclass" time="20.146" errors="0" timestamp="2024-01-10T00:22:04 IST" skipped="3">
  <testcase classname="Testng.firstclass" name="run" time="5.175">
    <skipped/>
  </testcase> <!-- run -->
  <system-out/>
  <testcase classname="Testng.firstclass" name="run" time="5.494">
    <skipped/>
  </testcase> <!-- run -->
  <system-out/>
  <testcase classname="Testng.firstclass" name="run" time="4.526">
    <failure message="expected [qwqw] but found [Google]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [qwqw] but found [Google]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Testng.firstclass.run(firstclass.java:20)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- run -->
  <system-out/>
  <testcase classname="Testng.firstclass" name="run" time="4.951">
    <skipped/>
  </testcase> <!-- run -->
  <system-out/>
</testsuite> <!-- Testng.firstclass -->
